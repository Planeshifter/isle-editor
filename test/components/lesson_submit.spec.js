// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import Provider from 'components/provider';
import Session from 'session';
import LessonSubmit from 'components/lesson-submit/main.js';


// VARIABLES //

const session = new Session( {}, true );


// TESTS //

it( 'renders a lesson submit button', () => {
	const button = renderer.create(
		<Provider session={session}>
			<LessonSubmit />
		</Provider>
	).toJSON();
	expect( button ).toMatchSnapshot();
});
