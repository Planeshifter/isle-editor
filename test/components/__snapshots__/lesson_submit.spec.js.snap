// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a lesson submit button 1`] = `
<span
  className="sk-cube-grid"
>
  <span
    className="sk-cube sk-cube1"
  />
  <span
    className="sk-cube sk-cube2"
  />
  <span
    className="sk-cube sk-cube3"
  />
  <span
    className="sk-cube sk-cube4"
  />
  <span
    className="sk-cube sk-cube5"
  />
  <span
    className="sk-cube sk-cube6"
  />
  <span
    className="sk-cube sk-cube7"
  />
  <span
    className="sk-cube sk-cube8"
  />
  <span
    className="sk-cube sk-cube9"
  />
</span>
`;
